app/dashboard/views.py

```diff
@@ -924,6 +927,282 @@ def save_search(request):
     return TemplateResponse(request, 'save_search.html', context)
 
 
+def funder_dashboard(request):
+    """ Render the funder dashboard"""
+    funder_bounties = request.user.profile.get_funded_bounties()
+
+    # TODO: Remove, it's just for testing now
+    funder_bounties = Bounty.objects.all()
+
+    active_done_expired_bounties = funder_bounties.filter_by_status(['active', 'done', 'expired'])
+    active_bounties = active_done_expired_bounties.filter_by_status(['active'])
+    done_bounties = active_done_expired_bounties.filter_by_status(['done'])
+    expired_bounties = active_done_expired_bounties.filter_by_status(['expired'])
+
+    contributor1 = {
+        "githubLink": "https://github.com",
+        "profilePictureSrc": "/static/avatar/NedelescuVlad",
+    }
+
+    # TODO: use done bounties to get the contributors
+    top_contributors = [contributor1]
+
+    # TODO: Fill in payout history data for the line chart
+    # Each one of these dictionaries is composed of 2 arrays that should have the same length.
+    # Each data[i] labels[i] represents a X Y coordinate (plot point)
+    payout_history_weekly = {
+        "data": [50, 105, 405, 150, 200, 300, 80, 305],
```

Comment:
you can use the `bounty.fulfillment_accepted_on` field to figure out the payout over time